sphinxgallery/gen_rst.py

```diff
@@ -445,7 +445,7 @@ def generate_file_rst(fname, target_dir, src_dir, gallery_conf, plot_gallery):
     Returns the set of functions/classes imported in the example.
     """
     base_image_name = os.path.splitext(fname)[0]
-    image_fname = '%s_%%03d.png' % base_image_name
+    image_fname = 'sphx_glr_%s_%%03d.png' % base_image_name
```

Comment:
Let's go for the sphx_glr prefix for now then and try to come up with a better long-term solution. 

It's easy enough to do a search and replace in the nilearn rst docs.